systems/overview/01-Introduction_and_Goals.md

```diff
@@ -1,5 +1,9 @@
 # Introduction and Goals
 
+Wikibase is an open-source software suite for creating collaborative knowledge bases, opening the door to the Linked Open Data web. It lets you store, manage and access structured data. Its two main sub-systems are [WikibaseRepo](../WikibaseRepo/01-Introduction_and_Goals.md) and [WikibaseClient](../WikibaseClient/01-Introduction_and_Goals.md).
+
+Wikibase is also the software behind [Wikidata](https://www.wikidata.org), a free, collaborative, multilingual, secondary database, collecting structured data to provide support for [Wikipedia](https://www.wikipedia.org), [Wikimedia Commons](https://commons.wikimedia.org/wiki/Main_Page), the other wikis of the [Wikimedia](https://www.wikimedia.org/) movement, and to anyone in the world. Its main comp
```

Comment:
> Its main comp

:mag:

Reply:
Ah, I know: "Its main components" moved up to the first paragraph about Wikibase and the lost sentence actually shows where my train of thought had stopped. when thinking about main components of Wikidata...

Fixed.